snapshot/lib/snapshot/detect_values.rb

```diff
@@ -42,6 +42,8 @@ def self.set_additional_default_values
           # Full list: ["iPhone 4s", "iPhone 5", "iPhone 5s", "iPhone 6", "iPhone 6 Plus", "iPhone 6s", "iPhone 6s Plus"]
           next if sim.name.include?("6s") # same screen resolution as iPhone 6, or iPhone 6s Plus
           next if sim.name.include?("5s") # same screen resolution as iPhone 5
+
+          # Filter AppleTv
```

Comment:
I agree on removing the comment, the line is quite self explanatory. 

But I think we should at least keep the empty line to improve its readability, otherwise it looks like an iPhone Filter attending to the surrounding code.